shared/utils/urls.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as urlsUtils from "./urls";
+
+describe("IsUrl Method", () => {
+  describe("invalid urls", () => {
+    it("should return false", () => {
+      expect(urlsUtils.isUrl("")).toEqual(false);
+      expect(urlsUtils.isUrl("#invalidurl")).toEqual(false);
+      expect(urlsUtils.isUrl("mailto:")).toEqual(false);
+      expect(urlsUtils.isUrl("://")).toEqual(false);
+    });
+  });
+});
+
+describe("isInternalUrl Method", () => {
+  it("should return false if empty string", () => {
+    expect(urlsUtils.isInternalUrl("")).toEqual(false);
+  });
+
+  it("should return true if starting with relative path", () => {
+    expect(urlsUtils.isInternalUrl("/drafts")).toEqual(true);
+  });
+});
+
+describe("isExternalUrl Method", () => {
+  it("should return false if empty url", () => {
+    expect(urlsUtils.isExternalUrl("")).toEqual(false);
+  });
+
+  it("should return false if internal url", () => {
+    expect(urlsUtils.isExternalUrl("/drafts")).toEqual(false);
+  });
+});
+
+describe("sanitizeUrl Method", () => {
+  it("should return undefined if not url", () => {
+    expect(urlsUtils.sanitizeUrl(undefined)).toEqual(undefined);
```

Comment:
[`toBeUndefined()`](https://jestjs.io/docs/expect#tobeundefined)? It's just handy.
```suggestion
    expect(urlsUtils.sanitizeUrl(undefined)).toBeUndefined();
```